### PR TITLE
AddFixed birthday pupup to check also "past" year birthday day

### PR DESCRIPTION
**Pull request by sharonco-matrix**
I forgot the year changes (oops)
Fixed to check the birthday passed not adding year not only month and day, as the patient that his birthday on the 31-Dec, we didn't saw the popup

### CODE REVIEW

File: library/reminder/src/BirthdayReminder.php [hunk 1]
```diff
@@ -57,6 +57,11 @@ public function isDisplayBirthdayAlert()
         return false;
     }
 
+    public static function __callStatic($name, $arguments)
```
Comment (bradymiller) [inline library/reminder/src/BirthdayReminder.php]: What's this method for?

File: library/reminder/src/BirthdayReminder.php [hunk 2]
```diff
@@ -57,6 +57,11 @@ public function isDisplayBirthdayAlert()
...
+    {
+        // TODO: Implement __callStatic() method.
```
Comment (sharonco-matrix) [inline library/reminder/src/BirthdayReminder.php]: Mistake :)

**Comment by bradymiller** [comment]
Code looks good and will bring into the codebase. Thanks for the fix! -brady

File: library/reminder/src/BirthdayReminder.php (hunk 2 above)
Comment (bradymiller) [reply library/reminder/src/BirthdayReminder.php]: no prob :)
Just brought in your fix. thanks!

**Pull request merged**